mysql/README.md

```diff
@@ -0,0 +1,233 @@
+treatment of fixed-length character types (marked c), and changes in storage
+size (marked s). We discuss these, as well as other limits and notes on
+schema conversion, in the following sections.
+
+### `DECIMAL` and `NUMERIC`
+
+Spanner does not support decimal types, so these are mapped to `FLOAT64`. For
+some decimal types (e.g. `DECIMAL(7, 3))` this mapping will preserve precision.
+But for others, the decimal type does not fit in `FLOAT64`; HarbourBridge
+generates a warning in such cases. In general, mapping `DECIMAL` to `FLOAT64`
+can be useful for evaluation purposes, but it is not recommended for production
+use. Note that in MySQL, NUMERIC is implemented as DECIMAL, so the remarks about DECIMAL apply equally to NUMERIC.
+
+### `TIMESTAMP` and `DATETIME`
+
+MySQL has two timestamp types: `TIMESTAMP` and `DATETIME`. Both provide microsecond resolultion, but neither
+actually stores a timezone with the data. The key difference between the two types is that MySQL converts `TIMESTAMP` values from the current time zone to UTC for storage, and back from UTC to the current time zone for retrieval. This does not occur `DATETIME` and data is returned
+without a timezone. For `TIMESTAMP`, timezone can be set by time zone offset parameter.
+
+Spanner has a single timestamp type. Data is stored as UTC (there is no separate
+timezone) Spanner client libraries convert timestamps to UTC before sending them
+to Spanner. Data is always returned as UTC. Spanner's timestamp type is
+essentially the same as `TIMESTAMP`, except that there is no analog of
+MySQL's timezone offset parameter.
+
+In other words, mapping MySQL `DATETIME` to `TIMESTAMP` is fairly
+straightforward, but care should be taken with MySQL `DATETIME` data
+because Spanner clients will not drop the timezone.
+
+### `CHAR(n)` and `VARCHAR(n)`
+
+The semantics of fixed-length character types differ between MySQL and
+Spanner. The `CHAR(n)` type in MySQL is right-padded with spaces. If a string
+value smaller than the limit is stored, spaces will be added to pad it out to
+the specified length. If a string longer than the specified length is stored,
+and the extra characters are all spaces, then it will be silently
+truncated. Moreover, trailing spaces are ignored when comparing two values. In
+constrast, Spanner does not give special treatment to spaces, and the specified
+length simply represents the maximum length that can be stored. This is close to
+the semantics of MySQL's `VARCHAR(n)`. However there are some minor
+differences. For example, even `VARCHAR(n)` has some special treatment of
+spaces: string with trailing spaces in excess of the column length are truncated
+prior to insertion and a warning is generated.
+
+### `SET`
+
+MySQL `SET` is a string object that can hold muliple values, each of which must be
+chosen from a list of permitted values specified when the table is created. `SET`
+is being mapped to `ARRAY<STRING>` spanner type. Validation of `SET` element values
```

Comment:
Suggest replacing:

`SET` is being mapped to `ARRAY<STRING>` spanner type.

by 

"`SET` is mapped to the Spanner type `ARRAY<STRING>`."